extensions/cms/src/cmsResource/tree/treeProvider.ts

```diff
@@ -68,7 +68,7 @@ export class CmsResourceTreeProvider implements TreeDataProvider<TreeNode>, ICms
 					server.ownerUri = undefined,
 						server.connection.options.password = '';
 				});
-				await this._appContext.apiWrapper.setConfiguration(toSaveCmsServers);
+				await this._appContext.cmsUtils.setConfiguration(toSaveCmsServers);
```

Comment:
generally Utility and Helper classes should only contain static methods so you don't need to create an instance to invoke methods.  if possible, I'd switch over to that pattern

Reply:
I needed the instance here because we cache the registered servers here as well as the cmsService which we get from mssql